packages/functionless-build-utils/src/compile-typedoc.ts

```diff
@@ -0,0 +1,37 @@
+import path from "path";
+import fs from "fs/promises";
+
+const pluginDocusaurus = require("docusaurus-plugin-typedoc").default;
+
+const rootDir = process.cwd();
+
+async function main() {
```

Comment:
add docs?

What is the benefit of this over doing it as part of docusuarus?

Reply:
It doesn't crash lmao